swagger-jaxrs/1.6.9.wso2v1/pom.xml

```diff
@@ -0,0 +1,101 @@
+    <description>
+        This bundle will export packages from swagger-core libraries of io.swagger
+    </description>
+    <url>http://wso2.org</url>
+
+    <distributionManagement>
+        <repository>
+            <id>wso2.releases</id>
+            <name>WSO2 internal Repository</name>
+            <url>https://maven.wso2.org/nexus/content/repositories/releases/</url>
+        </repository>
+    </distributionManagement>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <version>${google.guava.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.dataformat</groupId>
+            <artifactId>jackson-dataformat-yaml</artifactId>
+            <version>${fasterxml.jackson.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>io.swagger</groupId>
+            <artifactId>swagger-jaxrs</artifactId>
+            <version>${io.swagger.version}</version>
+            <optional>true</optional>
+        </dependency>
+    </dependencies>
+   
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <version>3.3.0</version>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Bundle-Name>${project.artifactId}</Bundle-Name>
+                        <Export-Package>
+                            io.swagger.*;version="${project.version}",                      
+                        </Export-Package>
+                        <Import-Package>
+                           javax.ws.rs,javax.ws.rs.core,javax.ws.rs.ext,
+                           com.fasterxml.jackson.*; version="${jackson.version.range}",
+                           com.google.common.*;version="${guava.import.version}",
+                           io.swagger.*; version="${io.swagger.version.range}",
+                           javax.servlet,
+                           javax.servlet.http,
+                           org.apache.commons.lang3;version="[3.12,4)", 
+                           org.reflections.*;version="${reflections.version.range}",
+                           org.slf4j; version="${slf4j.import.version.range}", 
+                        </Import-Package>
+                        <Private-Package>
+                        </Private-Package>
+                    </instructions>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+
+    <properties>
+        <google.guava.version>32.1.3-jre</google.guava.version>
+        <guava.import.version>[20.0.0,33.0.0)</guava.import.version>
+        <jackson.version.range>[2.7.0,3.0.0)</jackson.version.range>
```

Comment:
Shall we modify the guava version range to [31.0.0,33.0.0) and jackson range to [2.14.0,3.0.0). WDYT @AnuGayan ?

Reply:
+1 lets reduce the version range, since the swagger-jacrs was released with guava 31.0 and jackson 2.14